robotidy/transformers/NormalizeTags.py

```diff
@@ -0,0 +1,52 @@
+from robot.api.parsing import ModelTransformer
+from robot.api.parsing import Tags
+from robot.api.parsing import Token
+
+
+class NormalizeTags(ModelTransformer):
+    """
+    This transformer orders tags.
+    example usage:
+    robotidy --transform NormalizeTags:case=lowercase test.robot
+
+    Other supported cases: uppercase, titlecase. The default is lowercase.
+    See https://robotidy.readthedocs.io/en/latest/transformers/NormalizeTags.html for more examples.
+    """
+    CASE_FUNCTIONS = {'lowercase': str.lower, 'uppercase': str.upper, 'titlecase': str.title}
+
+    def __init__(self, case: str = 'lowercase'):
+        self.case = case
+        self.convert_case = self.CASE_FUNCTIONS[self.case]
```

Comment:
what if someone provides not supported case? for example:
```
NormalizeTags:case=idontexist
```

Reply:
handled similarly